srcpkgs/sumo/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'sumo'
+pkgname=sumo
+version=0.32.0
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config libgdal-tools"
+makedepends="fox-devel libgdal-devel libxerces-c-devel proj-devel"
+short_desc="SUMO (Simulation of Urban MObility) and included applications"
+maintainer="Warrior Graph <marques@larces.uece.br>"
+license="EPL-2.0"
+homepage="http://sumo.sourceforge.net/"
+distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-src-0.32.0.tar.gz"
+checksum=00753ca57a9911f0c99202505a6b05b1777168134842d7924fd827766642608a
+
```

Comment:
unnecessary extra blank lines

Reply:
I'm a little bit confused about the difference between **hostmakedepends** and **makedepends**. I hope it's right.